samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -232,7 +236,21 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                     label={teamsIdLabel}
                     required
                     placeholder={'Enter a Teams id'}
```

Comment:
Maybe we can also change the placeholder to 'Enter your Teams id (8:orgid:<UUID>)'

Reply:
Good idea I'll add that

Reply:
updated